Add common support requests for data.gov.uk (#1414)
alphagov/govuk-developer-docs: source/manual/data-gov-uk-2nd-line.html.md

2. A sysadmin user (e.g. 2nd line) can create an account for the new user.  Follow the instructions in [Assign users to publishers](#assign-users-to-publishers-setting-user-permissions) inputting the user's email address instead of their username.  An invite email is generated and sent to the publisher.

> An account should only be created if the user intends to publish on data.gov.uk.  Previously users have been able to register accounts to follow publishers.  This is no longer offered.

### Create a publishing organisation
1. Login to [CKAN][ckan] as a sysadmin user (credentials are available in the `govuk-secrets` password store, under `datagovuk/ckan`).
2. Click the 'Publishers' button in the header.
3. Click 'Add Publisher' and complete the form.
4. Ensure that a publishing user has been assigned as an 'Admin' for this organisation (this would normally be the person who makes the request, so they can then add further 'Editor' users without needing to contact support).

> 

Reviewer: Should we also put this warning on the "create publishing org" section since a user is being assigned admin rights there?
```suggestion

> Check the authenticity of a request before adding a user as a publisher (i.e. make sure they actually belong to the department they want to publish for, bearing in mind that some parent organisations may publish on behalf of child organisations, e.g. BEIS can publish for the Civil Nuclear Police Authority).

```

4. Click the 'Manage' button to get the status.
5. A summary of the current status will be shown.  Individual runs (and the error messages logged) can be access from the 'Jobs' tab.

### Restart a harvester
1. Login to [CKAN][ckan] as a sysadmin user (credentials are available in the `govuk-secrets` password store, under `datagovuk/ckan`).
2. Navigate to the relevant harvester (use the 'Harvest' button in the header).
3. You will see a list of the datasets imported by this harvest source.
4. Click the 'Manage' button.
5. If the harvester is currently running, click the 'Stop' button to stop it.  Once it has stopped, or if it is not currently running, click the 'Reharvest' button.  You will know if the harvester is running because the 'Reharvest' button will be disabled.

## Organogram Publishing
Organograms are files that allow the people structure of an organisation to be visualised.  They are split into two files: one for senior staff (grades SCS1, SCS2 and SCS3, or equivalent) and another for junior staff (all other grades).  The senior staff file is more detailed than the junior staff file, with staff names included for posts classified as grades SCS2 and SCS3.

> Organograms are unique in data.gov.uk since we actually host the data, rather than linking to an external resource.  The data files are hosted in a S3 bucket (named `datagovuk-integration-ckan-organogram`, `datagovuk-staging-ckan-organogram` or `datagovuk-production-ckan-organogram`, depending on the environment).

Reviewer: When talking to devs about might be better to avoid using the word "unique"?
```suggestion
 Organograms are the only datasets in data.gov.uk where we host the data itself rather than linking to an external resource.  The data files are hosted in a S3 bucket (named `datagovuk-integration-ckan-organogram`, `datagovuk-staging-ckan-organogram` or `datagovuk-production-ckan-organogram`, depending on the environment).
```
